src/ralph/dhcp/views.py

```diff
@@ -135,6 +136,11 @@ def get_last_modified(self, networks):
         IP (DHCP entry), ethernet.
         """
         last_items = []
+
+        try:
+            last_items.append(Deployment.objects.latest('modified').modified)
```

Comment:
shouldn't we filter deployments for this particular network(s) only? :) It'll probably be not so easy with current models